encryptedlogging/src/androidTest/kotlin/com/automattic/encryptedlogging/release/ReleaseStack_EncryptedLogTest.kt

```diff
@@ -61,23 +52,23 @@ internal class ReleaseStack_EncryptedLogTest {
     }
 
     @After
-    fun tearDown() {
+    fun tearDown() = runTest {
         // Reset the 'uploadState' of 'EncryptedLogStore' so that both tests can run, because it is now a singleton.
         encryptedLogStore.uploadState.value = null
         cleanSharedPreferencesState()
-        WellSql.delete(EncryptedLogModel::class.java).execute()
+        database.encryptedLogDao.deleteEncryptedLogs()
```

Comment:
Minor, but I think we could close db here as well

```suggestion
        database.encryptedLogDao.deleteEncryptedLogs()
        database.close()
```

Reply:
Thanks for this comment @wzieba , I actually tried that but it doesn't work because when the database is closed (after the first test, which succeeds), then that makes any subsequent test(s) fail. This is different to the in-memory database on unit tests... 🤷

Reply:
ah I see 👍 `database` here is from Singleton, which survives the JUnit singular execution. I guess it's not a big deal, but I wonder what would happen if you'd cancel the test execution in the middle? Will `@After` method be called and will DB be cleared?

Reply:
> ...what would happen if you'd cancel the test execution in the middle?

How and why would you do that @wzieba? 🤔

> Will @After method be called and will DB be cleared?

When you cancel the execution, even if the `@After` method isn't called, and the database isn't cleared, how would that be a problem to us? 🤔

Reply:
> How and why would you do that @wzieba? 🤔

Just stop the process, Gradle crash, disconnect the phone - many possibilities.

> how would that be a problem to us? 🤔

The database will contain events from previous executions, which might affect current test run.